config/config.dist.php

```diff
@@ -1,10 +1,13 @@
 <?php
+
+namespace Imbo\MetadataSearch;
+
 return [
     'eventListeners' => [
         'metadata' => [
             'listener' => 'Imbo\MetadataSearch\EventListener\MetadataOperations',
```

Comment:
Hmm. I think I ever-so-slightly prefer the alternative (`new EventListener\...`). But I'd be okay with either.